chrononaut/change_info.py

```diff
@@ -25,24 +25,19 @@ class ChangeInfoMixin(object):
     """
 
     @classmethod
-    def _capture_change_info(cls):
-        """Capture the change info for the new version. By default calls:
+    def _capture_user_info(cls):
```

Comment:
We treat it as a mixin. Not sure if it's worth being a separate class though in practice. It could be moved to `RecordChanges` class in my opinion. @boydgreenfield thoughts?

Reply:
If there is a plan to allow users to override it than it is better to leave it as a class method. But an underscore prefix suggest that it is a private API.

Reply:
I think this is "you can override if you know what you're doing" (so us). And a private class method is a good balance here.